src/ToDo/Business/TaskService.cs

```diff
@@ -38,4 +38,14 @@ public async Task DeleteAsync(ToDoTask task, CancellationToken ct)
 
 		_messenger.Send(new EntityMessage<ToDoTask>(EntityChange.Delete, task));
 	}
+
+	public async ValueTask<IImmutableList<ToDoTask>> GetByFilterAsync(string displayName, CancellationToken ct)
```

Comment:
We shouldnt have 2 methods in the service layer. LEt's put an optional filter in the GetAll and depending if the value is et or not, let's call the proper endpoint.